lib/acmex/client.ex

```diff
@@ -4,20 +4,39 @@ defmodule Acmex.Client do
   use GenServer
 
   alias Acmex.{Crypto, Request}
-  alias Acmex.Resource.{Account, Challenge, Directory, Order}
-
-  def init(keyfile: keyfile) do
-    with true <- File.exists?(keyfile),
+  alias Acmex.Resource.{Account, Authorization, Challenge, Directory, Order}
+
+  def init(opts \\ []) do
+    keyfile = Keyword.get(opts, :keyfile)
+    key = Keyword.get(opts, :key)
+
+    with {:ok, jwk} <-
+           (cond do
```

Comment:
What do you think about extract this `cond` from here? I recommend a private function or a var outside the `with`.